.ci_support/linux_64_.yaml

```diff
@@ -9,6 +9,8 @@ cxx_compiler:
 cxx_compiler_version:
 - '12'
 docker_image:
-- quay.io/condaforge/linux-anvil-cos7-x86_64
+- quay.io/condaforge/linux-anvil-alma-x86_64:8
```

Comment:
Please don't manually touch these files. You will need to increase the `c_stdlib_version` for this to work.

Reply:
I reverted this change.